src/components/views/Misc/About.jsx

```diff
@@ -118,4 +119,19 @@ const About = () => {
   );
 };
 
+const About = () => {
+  const enableAbout = useAppSelector(
+    (state) => state.featureFlagState["enableAbout"]
+  );
+
+  return (
+    <div>
+      {enableAbout === "Enabled" ? (
```

Comment:
Don't use magic keywords `"Enabled"` here. Use `FFState.Enabled` instead. Same in `FAQ.jsx` and in `Nav.tsx` (`FeatureFlagElement`)